core_lib/managers/soundmanager.cpp

```diff
@@ -90,7 +90,7 @@ Status SoundManager::loadSound( Layer* soundLayer, int frameNumber, QString strS
         soundLayer->addKeyFrame( frameNumber, key );
     }
 
-    if ( !key->fileName().isEmpty() )
+    if ( key->fileName().isEmpty() )
```

Comment:
Just think of it. Because in this function it's loading a new audio file, so the SoundClip's file path should be empty, we don't want to overwrite an existing sound clip. So..the original code is correct. I'll make a change and put a comment on it.

Reply:
Aaah riight yes well that make sense.